### ISSUE TITLE
Show string implementation

### ISSUE THREAD

**Issue by joneshf** [jsverify/jsverify#69]
When showing a string it just returns a raw string. However, when you want to use the show implementation in a larger data structure, it doesn't really work too well. For instance

```javascript
var Ident = function(a) {
  this.val = a;
}
var IdentShow = function(a) {
  return function(id) {
    return "Ident(" + a.show(id.val) + ")";
  };
}
```

If you make an `Ident string` when it goes to show it, sometimes it will show `Ident()`, if the generated string was the empty string for instance. What seems more natural would be to see `Ident("")` or something indicating that the string was empty.

It seems like it is enough to just change these lines:

https://github.com/jsverify/jsverify/blob/67a97152fc91b1d4ace6b2d8463e06d132dad717/lib/primitive.js#L252
https://github.com/jsverify/jsverify/blob/67a97152fc91b1d4ace6b2d8463e06d132dad717/lib/primitive.js#L263
https://github.com/jsverify/jsverify/blob/67a97152fc91b1d4ace6b2d8463e06d132dad717/lib/primitive.js#L273
https://github.com/jsverify/jsverify/blob/67a97152fc91b1d4ace6b2d8463e06d132dad717/lib/primitive.js#L287
https://github.com/jsverify/jsverify/blob/67a97152fc91b1d4ace6b2d8463e06d132dad717/lib/primitive.js#L296

to be like this one:

https://github.com/jsverify/jsverify/blob/67a97152fc91b1d4ace6b2d8463e06d132dad717/lib/primitive.js#L309-L311

Thoughts?

**Comment by phadej** [comment]
Yeah, sounds good. Could you check, whether one could just change `show.def` definition to use `JSON.stringify`?

Just wondering if the default `toString` is actually usable that much...

**Comment by joneshf** [comment]
Closed by #73. Thanks!
